client/flutter/lib/l10n/intl_es.arb

```diff
@@ -1,23 +1,26 @@
 {
-    "worldHealthOrganization": "",
-    "learnMore": "",
-    "protectYourself": "",
-    "checkYourHealth": "",
-    "localMaps": "",
-    "shareTheApp": "",
-    "healthCheck": "",
-    "feelingDistressed": "",
-    "washHands": "",
-    "cougningAndSneezing": "",
-    "throwAwayTissue": "",
-    "washHandsFrequently": "",
-    "socialDistancing": "",
-    "seekMedicalCare": "",
-    "infoForEveryone":"",
-    "infoForParents":"",
-    "childStress":"",
-    "childLove":"",
-    "childDistance":"",
-    "childRoutines":"",
-    "childFacts":""
+    "worldHealthOrganization": "Organización Mundial de la Salud",
+    "learnMore": "Aprenda más",
+    "protectYourself": "Protéjase",
+    "checkYourHealth": "Consulte su estado de salud",
+    "localMaps": "Mapas Locales",
+    "shareTheApp": "Compartir la aplicación",
+    "healthCheck": "Consulta de salud",
+    "feelingDistressed": "Se siente angustiado/a?",
+    "washHands": "Lávese las manos a menudo con agua y jabón",
+    "cougningAndSneezing": "Cuando tosa o estornude, tápese la boca y la nariz con un pañuelo, o con su codo",
+    "throwAwayTissue": "Eche a la papelera inmediatement cualquier pañuela después de su uso",
```

Comment:
~~inmediatament~~ -> inmediatamente

Reply:
can you clarify these comments? not sure I understand exactly what they're indicating

Reply:
welp, this is embarrassing - Google Chrome automatically translated these comments for me without my realizing. this makes a lot more sense now! you can disregard my comment